### ISSUE TITLE
rvd log files are rotated when `sudo rvc reload`

### ISSUE THREAD

**Issue by erikbor** [riboseinc/rvc#211]
`rvd` log files are automatically rotated when `sudo rvc reload`:

```
$ ls -la /var/log/rvd
total 56
drwxr-xr-x   4 root  wheel    136 Nov 27 10:52 .
drwxr-xr-x  65 root  wheel   2210 Nov 27 09:00 ..
-rw-------   1 root  wheel  22996 Nov 27 10:52 rvd.log
-rw-------   1 root  wheel     75 Nov 27 10:52 rvd.log.0
$ sudo rvc reload
Sending reload signal to rvd process '60543' has succeeded.
$ ls -la /var/log/rvd
total 16
drwxr-xr-x   4 root  wheel   136 Nov 27 10:52 .
drwxr-xr-x  65 root  wheel  2210 Nov 27 09:00 ..
-rw-------   1 root  wheel  1144 Nov 27 10:52 rvd.log
-rw-------   1 root  wheel    75 Nov 27 10:52 rvd.log.0
$ 
```

This makes it very hard to troubleshoot because `tail -f` will stop when reloading.

Can you stop the rotation/overwriting when reloading and append to the existing log file? Thanks

**Comment by jjr840430** [comment]
Fixed by #215

**Comment by ronaldtse** [comment]
Confirmed.
